website/oauth/views.py

```diff
@@ -50,4 +50,8 @@ def oauth_callback(service_name, auth):
         user.external_accounts.append(provider.account)
         user.save()
 
+    oauth_complete.send({
```

Comment:
The first argument should be the "sender" of the signal; in this case, that probably means the `provider`.

The subsequent argument are just additional data. So how about

```python
oauth_complete.send(provider, user)
```

This has the added benefit of enforcing the signature of listeners (e.g. an error will be raised if a listener doesn't include both the provider and user arguments)